docs/001_develop/03_client-capabilities/011_criteria/02_criteria-builder.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Criteria builder'
+sidebar_label: 'Criteria builder'
+id: criteria-builder
+keywords: [foundation, ui, comms, criteria, filtering]
+tags:
+  - connected
+  - ui
+  - comms
+  - criteria
+  - filtering
+---
+
+The `CriteriaBuilder` is a utility class designed to build objects used in creating query criteria in a Groovy expression format. These criteria can be used in requests to backend services.

```

Comment:
Add a link to the API doc for http://localhost:8080/develop/client-capabilities/criteria/docs/api/foundation-criteria.criteriabuilder/ in this page somewhere